.travis.yml

```diff
@@ -1,6 +1,20 @@
 language: python
 python:
-  - "3.5.1"
-
+- 3.5.1
 script:
-  - python examples/validate.py
+- python examples/validate.py
+before_deploy:
+- python find-latest-schemas.py schemas/ schemas-latest/
```

Comment:
The script find-latest-schemas.py needs to be copied from the Eiffel core repo I assume for this to work.